core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -109,24 +113,28 @@ var ExplorationEditorSettingsTab = function() {
       closePreviewSummaryButton);
     await waitFor.invisibilityOf(
       explorationSummaryTile, 'Summary Tile takes too long to disappear');
-    await action.click('Neutral element', neutralElement);
+    await action.click('Neutral element', neutralElement, true);
```

Comment:
ditto: masked element. Throughout this PR, it looks like you've added a lot of `elementIsMasked=true` to `action.click` calls in cases where we don't expect the element to be masked. Please either undo these changes or explain why elements are masked now

Reply:
removed